jwst/lib/tests/test_set_telescope_pointing.py

```diff
@@ -590,7 +591,8 @@ def test_add_wcs_method_full_siafdb(eng_db_ngas, data_file, tmp_path):
 
         with datamodels.open(DATA_PATH / expected_name) as expected:
             for meta in METAS_EQUALITY:
-                assert model[meta] == expected[meta]
+                #assert model[meta] == expected[meta]
+                assert np.isclose(model[meta], expected[meta], atol=1e-13)
```

Comment:
This will not work because the `meta` contains values of type `str` and those cannot be compared with `isclose`. 
One way to do this is to move all numerical values from `METAS_EQUALITY` to `METAS_ISCLOSE` and leave the string values in `METAS_EQUALITY` where they can be directly compared as before.

Reply:
Yep, of course. Why didn't I see that?  Latest commit does different comparisons depending on datatype of arguments. Cross your fingers ...